### ISSUE TITLE
[Adaptive Cards - Customer Credit Request]: All the edit form fields are not associated with their respective labels.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7445]
https://microsoft.visualstudio.com/OS/_workitems/edit/38275598

NOTE: bug was created 2/23/2022. Goes out of SLA after 120 days (aka 6/23/2022)

**Comment by carlos-zamora** [comment]
Closing. Issue is that the JSON still isn't using `label` property even after updating to AC v1.3
